docs/proposals/cli/README.md

```diff
@@ -0,0 +1,143 @@
+For the v0 of orka, only some services are available, which you can see below:
+
+### Provided services
+
+- single tenant  
+- single node  
+- single workload kind (containers)  
+
+
+### Out of scope services (for v0)
+
+- Ressource limitation  
+- Multi tenant
+- Multi workload (VM, containers, bare metal)
+- Scalabilty
+- High Availability
+- Upgradable
+- Monitoring
+- Per tenant networks
+
+
+## High Level Architecture
+
+As the cluster CLI, `orkactl` communicates with the Cluster REST API to send requests to it and retrieve cluster information back from it.
+
+```mermaid
+---
+title: "Orka: High-level Architecture"
+---
+flowchart LR
+    cli[CLI]
+
+    subgraph Cluster
+        subgraph Control Plane
+            api[Cluster API]
+            ctr[Controller]
+            sch[Scheduler]
+        end
+
+        subgraph Nodes
+            subgraph Node1
+                a1[Agent]
+            end
+            subgraph Node2
+                a2[Agent]
+            end
+            subgraph Node3
+                a3[Agent]
+            end
+        end
+
+    end
+
+    cli <--> api
+    api <--> ctr
+    ctr <--> sch
+
+    a1 & a2 & a3 --> sch
+
+    a1 ~~~ a2 ~~~ a3
+
+```
+
+
+## Transport Protocol
+
+As a CLI, our component interface will communicate to the ClusterAPI using the HTTPS Protocol.
+
+Our CLI will contact the API through the following routes:
+
+- `GET /workload` returns all workloads
+- `GET /instance` returns all instances
+- `GET /workload/:id` returns workload of id `:id`
+- `GET /instance/:id` returns instances of id `:id`
+- `POST /workload` creates a new workload
+- `POST /instance` creates a new instance
+- `DELETE /workload/:id` deletes an existing workload of id `:id`
+- `DELETE /instance/:id` deletes an existing instance of id `:id`
+
+
+## CLI
+
+`orkactl` provides the following set of commands:
+
+**Optional arguments** are represented between `[]`
+
+**Parameter arguments** are represented between `<>`
+
+- `orkactl config set <configResource> <value>`
+
+- `orkactl config get <configResource>`
+
+- `orkactl create workload -f work.yaml`
+
+- `orkactl delete workload <workload-id>`
+
+- `orkactl create instance <workload-id>`
+
+- `orkactl delete instance <instance-id>`
+
+- `orkactl get workload [--id <workload-id>]`
+
+- `orkactl get instance [--id <instance-id>]`
+
+
+## Workload description
+
+The workload will be presented in a yaml file.
+
+The yaml will need to follow this pattern: 
+
+
+```yaml
+version: 1                     # Default to last supported version 
+workload:
+    kind: "container"          # Only "container" for v0
+    name: postgres
+    environment:
+        - key: value
+        - key2: value2
+        - keyX: valueX
+
+    registry: ghcr | dockerhub # Default to dockerhub, optional
+    image: postgres:15
+    port: "EXPOSED PORT:INTERNAL PORT"
+
+```
+
+## Config
+
+`orkactl` users can configure the CLI through a configuration file, located by default at `$HOME/.orka/config.yaml`.
```

Comment:
Please use the standard XDG user directory for storing `orkactl`'s configuration.
```suggestion
`orkactl` users can configure the CLI through a configuration file, located by default at `$HOME/.config/orka/config.yaml`.
```